src/components/LoanCards/BorrowerInfo/BorrowerInfoHeader.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+	<div class="borrower-info-header">
+		<router-link
+			:to="`/lend/${loanId}`"
+			class="name"
+			v-kv-track-event="['Lending', 'click-Read more', 'Name', loanId, 'true']"
+			v-if="name"
+		>
+			<span
+				@click="$emit('track-loan-card-interaction', {
+					interactionType: 'viewBorrowerPage',
+					interactionElement: 'borrowerName'
+				})"
+			>
+				{{ name }}
+			</span>
+		</router-link>
+		<div class="country" v-if="country">{{ country }}</div>
```

Comment:
This will likely need to come out of here too at some point. Pretty sure the mini loan cards show country + activtiy under the title. As you work on the list card, I would move the borrower info wrapper div into your card and then include each item directly, preferably as a "dumb" components that just receive the single value for the name...or country...or activity. This as opposed to combining them as a group which is not versatile.

Reply:
Interesting, I like the idea here. Let's discuss this further in our sync up.